src/legacy/core_plugins/metrics/server/lib/vis_data/get_annotations.js

```diff
@@ -30,21 +31,22 @@ function validAnnotation(annotation) {
 
 export default async (req, panel) => {
   const { callWithRequest } = req.server.plugins.elasticsearch.getCluster('data');
+  const timeout = getEsShardTimeout(req);
   const bodies = panel.annotations
     .filter(validAnnotation)
     .map(annotation => {
-
       const indexPattern = annotation.index_pattern;
       const bodies = [];
+      const body = {
+        ...buildAnnotationRequest(req, panel, annotation),
+        timeout
```

Comment:
Hm...  it's a working code but probably it will be better if buildAnnotationRequest return us body with  the 'timeout'. Code will be a more readable. 

@timroes  @markov00  Any ideas?

Reply:
Yes it can be moved inside the buildAnnotationRequest